physionet-django/templates/footer.html

```diff
@@ -1,18 +1,60 @@
-<!-- This footer template is from the bootstrap 'blog' example -->
 {% load static %}
 <link rel="stylesheet" type="text/css" href="{% static 'custom/css/footer.css' %}"/>
 
-<footer class="blog-footer">
-  <div class="container">
-    <div class="row row-centered">
-      <div class="col-xs-10 center col-centered">
-        <p>{{ SITE_NAME }} is a repository of freely-available medical research data, managed by the {{ FOOTER_MANAGED_BY }}.</p>
-        <p>Supported by the {{ FOOTER_SUPPORTED_BY }}.</p>
-        {% if FOOTER_ACCESSIBILITY_PAGE %}
-          <p>For more accessibility options, see {{ FOOTER_ACCESSIBILITY_PAGE|safe }}.</p>
-        {% endif %}
-        <p><a href="#">Back to top</a></p>
+<div class="modern-ui">
+  <footer class="footer w-100 secondary-bg py-5">
+    <div class="container my-0 py-0 py-md-5">
+      <img src="{% static 'images/logo-white.png' %}" width="180px" alt="PhysioNet Logo" class="pb-2" />
+      <div class="footer__columns light-text">
+        <div>
+          <p class="dark-light-text mb-2">PhysioNet is a repository of freely-available medical research data, managed by the MIT Laboratory for Computational Physiology.</p>
```

Comment:
The codebase is used by other repositories (not just Physionet) so content like "PhysioNet is a repository of freely-available medical research data, managed by the MIT Laboratory for Computational Physiology" should not be hardcoded. I would suggest moving all of this text to environment variables (and actually I think some of the variables already exist, e.g.:

- `FOOTER_ACCESSIBILITY_PAGE`
- `SITE_NAME`
- `FOOTER_SUPPORTED_BY`
- `FOOTER_MANAGED_BY`